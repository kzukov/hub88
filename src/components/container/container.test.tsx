import React from "react";
import Container from "./container";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("<Container />", () => {
  test("matches snapshot", () => {
    const { asFragment } = render(<Container>children</Container>);

    expect(asFragment()).toMatchSnapshot();
  });
});
