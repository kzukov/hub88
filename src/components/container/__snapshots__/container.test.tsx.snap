// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    children
  </div>
</DocumentFragment>
`;
